code/modules/mob/living/carbon/human/species.dm

```diff
@@ -654,6 +654,11 @@
 		var/obj/item/weapon/tank/jetpack/J = H.back
 		if(J.allow_thrust(0.01, H))
 			hasjetpack = 1
+	else if(istype(H.wear_suit,/obj/item/clothing/suit/space/hardsuit))
```

Comment:
you can probably replace this and the other jetpack check with get_type_in_all_contents() (since that should check the human, the human's contents, and the contents of the human's contents, finding both worn and internal jetpacks)